reinforcement_learning/README.md

```diff
@@ -0,0 +1,20 @@
+# Reinforcement Learning using Cloud ML Engine
+
+We present examples of running reinforcement learning (RL) algorithms on GCP
+using Cloud ML Engine, specifically, we provide implementations of [DDPG](https://arxiv.org/abs/1509.02971) and [TD3](https://arxiv.org/abs/1802.09477), train and tune their hyper-parameters in the [BipedalWalker](https://gym.openai.com/envs/BipedalWalker-v2) environment.
```

Comment:
if there are any advantages of using CMLE (e.g. managed hptuning), briefly list them here in the readme.

Reply:
will consider that, thanks for the advise